docs/html/development/contributing.rst

```diff
@@ -39,8 +39,11 @@ separately, as a "formatting cleanup" PR, if needed.
 Automated Testing
 =================
 
-All pull requests and merges to 'master' branch are tested using `Travis CI`_
-and `Appveyor CI`_ based on our `.travis.yml`_ and `.appveyor.yml`_ files.
+All pull requests and merges to 'master' branch are tested using `Travis CI`_,
+`Azure DevOps CI`_ and `GitHub Actions`_ based on our `.travis.yml`_,
```

Comment:
```suggestion
`Azure DevOps`_ and `GitHub Actions`_ based on our `.travis.yml`_,
```

Reply:
IMO directly saying "Azure Pipelines" would be better here.

Reply:
So "Azure Devops" is the entire devops suite MSFT provides for testing (https://azure.microsoft.com/en-in/services/devops/) , and "Azure Pipelines" is the product we use to run our CI/CD pipelines (https://azure.microsoft.com/en-in/services/devops/pipelines/)

Perhaps "Azure Pipelines" is better since we have a folder named "azure-pipelines" in our code. So If I am renaming this, should I also change the link?

Reply:
Okay, I have gone ahead and changed "Azure Devops" to "Azure Pipelines" , and updated the link accordingly